src/clparser_perftest.cc

```diff
@@ -18,7 +18,7 @@
 #include "clparser.h"
 #include "metrics.h"
 
-int main(int argc, char* argv[]) {
+int mainUTF8(int argc, char* argv[]) {
```

Comment:
Please note that it would be better if you not rename the main function to mainUTF8. Windows is the only platform that has this problem, so it's the only platform that should pay a cost.

Have wmain call regular main, instead of requiring all platforms needing to call a second function.

Reply:
This seems to trigger some unwanted behaviour. cl is unable to handle both wmain and main in the same file and defaults to main...
How about changing the entrypoint to something like

```
#ifdef _WIN32
int wmain(int argc, wchar_t** wargv){
  char** argv = convertCommandline(wargv)
#else
int main(int argc, char** argc){
#endif
```

Reply:
Sounds good :) And it's even shorter.

Reply:
Sounds great